Sources/SwiftDocC/Semantics/Metadata/CallToAction.swift

```diff
@@ -79,6 +81,7 @@ public final class CallToAction: Semantic, AutomaticDirectiveConvertible {
 
     /// The computed label for this Call to Action, whether provided directly via ``label`` or
     /// indirectly via ``purpose``.
+    @available(*, deprecated, renamed: "buttonLabel(for:)")
     public var buttonLabel: String {
         if let label = label {
```

Comment:
should this internally call `buttonLabel(for: nil)`?

Reply:
Good idea: https://github.com/apple/swift-docc/pull/566/commits/c83c4a35f0b53a508772e1e4f041722141516564.